src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -174,6 +169,98 @@ const MissingData = styled.div`
   gap: 20px;
 `
 
+export const AboutContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  padding: 24px 0px;
+`
+
+const TokenDescriptionContainer = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 100%;
+  max-height: fit-content;
+`
+
+const TruncateDescriptionButton = styled.div`
+  color: ${({ theme }) => theme.textSecondary};
+  font-weight: 400;
+  font-size: 14px;
+  flex: none;
+  padding: 12px 0px;
+
+  &:hover,
+  &:focus {
+    color: ${({ theme }) => darken(0.1, theme.textSecondary)};
+    cursor: pointer;
+  }
+`
+
+type TokenDetailDataProps = {
+  description: string | null
+  homepageUrl: string | null
+  twitterName: string | null
```

Comment:
string?

Reply:
actually can leave this, i didn't realize this was the data type not the props.

Reply:
sgtm